### PR TITLE
Add new css handles

### PR DESCRIPTION
**Pull request by kaisermann**
#### What problem is this solving?

Add missing css handles to the pickup availability section.

Related to: https://app.clubhouse.io/vtex/story/32542/add-css-handles-to-pickup-availability

#### How should this be manually tested?

[Workspace](url)

#### Checklist/Reminders

- [x] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [x] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Screenshots or example usage

image

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
✔️| New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- New css handles to pickup availability container.
```
Comment (klzns) [inline CHANGELOG.md]:
```suggestion
- New CSS Handles to pickup availability container.
```

File: react/components/ContainerStateSelector.tsx [hunk 2]
```diff
@@ -71,17 +75,18 @@ const ContainerStateSelector: FC<Props> = ({ showSelectSkuMessage, googleMapsKey
     )
   }
 
+
```
Comment (klzns) [inline react/components/ContainerStateSelector.tsx]:
```suggestion
```

File: react/components/ContainerStateSelector.tsx [hunk 3]
```diff
@@ -71,17 +75,18 @@ const ContainerStateSelector: FC<Props> = ({ showSelectSkuMessage, googleMapsKey
...
-      <div className="c-muted-2 t-body mb1 ml3">
+    <div className={`${handles.innerContainer} flex flex-column`}>
+      <div className={`${handles.availabilityHeader} c-muted-2 t-body mb1 ml3`}>
         <FormattedMessage id="store/pickup-availability.availability-header" />
       </div>
-      <div className="c-on-base t-body mb1 ml3 mt3">
+      <div className={`${handles.availabilityMessage} c-on-base t-body mb1 ml3 mt3`}>
         <FormattedMessage id="store/pickup-availability.available-for-pickup" />
       </div>
       <div>
```
Comment (klzns) [inline react/components/ContainerStateSelector.tsx]: Missing one here

File: yarn.lock [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```
Comment (klzns) [inline yarn.lock]:
```suggestion
```

**Review by klzns — CHANGES_REQUESTED** [review]

File: react/components/ContainerStateSelector.tsx [hunk 5]
```diff
@@ -72,16 +90,18 @@ const ContainerStateSelector: FC<Props> = ({ showSelectSkuMessage, googleMapsKey
...
         <FormattedMessage id="store/pickup-availability.availability-header" />
       </div>
-      <div className="c-on-base t-body mb1 ml3 mt3">
+      <div
+        className={`${handles.availabilityMessage} c-on-base t-body mb1 ml3 mt3`}
+      >
         <FormattedMessage id="store/pickup-availability.available-for-pickup" />
       </div>
-      <div>
+      <div className={`${handles.availabilityButtonContainer}`}>
```
Comment (klzns) [inline react/components/ContainerStateSelector.tsx]: I'm proposing this change so if we add another button we don't get stuck

```suggestion
      <div className={`${handles.availabilityChooseStoreButton}`}>
```

File: react/components/ContainerStateSelector.tsx [hunk 6]
```diff
@@ -72,16 +90,18 @@ const ContainerStateSelector: FC<Props> = ({ showSelectSkuMessage, googleMapsKey
...
         <Button onClick={() => setShowForm(true)} variation="tertiary">
-          <div className="t-body nh5">
+          <div className={`${handles.availabilityAction} t-body nh5`}>
```
Comment (klzns) [inline react/components/ContainerStateSelector.tsx]:
```suggestion
          <div className={`${handles.availabilityChooseStoreText} t-body nh5`}>
```

File: CHANGELOG.md (hunk 1 above)
Comment (klzns) [inline CHANGELOG.md]:
```suggestion
- New CSS Handles to pickup availability container.
```

**Review by klzns — CHANGES_REQUESTED** [review]

File: react/components/ContainerStateSelector.tsx (hunk 5 above)
Comment (kaisermann) [reply react/components/ContainerStateSelector.tsx]: neaaat

**Review by klzns — APPROVED** [review]

**Review by iaronaraujo — APPROVED** [review]

**Pull request merged**